source/index.txt

```diff
@@ -182,10 +182,15 @@ For documentation about previous releases, see the `legacy documentation <https:
      - Active development
      - New feature development
 
-   * - mongocxx 4.0.x
+   * - mongocxx 4.1.x
      - Stable
      - Bug fixes only
      - Current stable C++ driver release 
+
+   * - mongocxx 4.0.x
+     - Stable
+     - Bug fixes until ????
```

Comment:
```suggestion
     - None
```

Quoting [Policy: Drivers Maintenance & Compatibility](https://docs.google.com/document/d/1Y3aNiHP_g7uqgLLdRnEDQRLl6yz45PVRDoUDsjMkRD0/edit?tab%3Dt.0%23heading%3Dh.hu2cddix0mil&sa=D&source=docs&ust=1746459284711434&usg=AOvVaw3Up3PZMEw98CpMSgIZcEZX)

> Relevant bug fixes will be backported from the current major version to the previous major version for a period of one year after a new major version is released.

> Patch versions (semver Z) of a minor version will stop when the next minor version (semver Y) is released.

3.11 is supported with bug fixes for one year after 4 is released. But with 4.1 released, 4.0 is no longer actively maintained.